docs/Reference/Server.md

```diff
@@ -351,9 +349,12 @@ The possible values this property may have are:
         ```
       Any user-supplied serializer will override the default serializer of the
       corresponding property.
-+ `loggerInstance`: a custom logger instance. The logger must conform to the
-  Pino interface by having the following methods: `info`, `error`, `debug`,
-  `fatal`, `warn`, `trace`, `child`. For example:
+
+### `loggerInstance`
+
+a custom logger instance. The logger must be a pino instance or  conform to the
+Pino interface by having the following methods: `info`, `error`, `debug`,
+`fatal`, `warn`, `trace`, `child`. For example:
```

Comment:
Please follow the same documentation format as the other options.

Reply:
lmk if 02eba50 does what you want, happy to change it